static/oas/v1/openapi.yml

```diff
@@ -79,3 +79,7 @@ components:
       type: http
       scheme: bearer
       bearerFormat: jwt
+
```

Comment:
Do we need to add 
```
components:
  securitySchemes:
    bearerAuth:            # arbitrary name for the security scheme
      type: http
      scheme: bearer
      bearerFormat: JWT    # optional, arbitrary value for documentation purposes
```
https://swagger.io/docs/specification/authentication/bearer-authentication/

Reply:
removing since functionality has not been implemented, but will need in future